fireworks_poe_bot/__main__.py

```diff
@@ -76,82 +76,40 @@ def main(args=None):
             else:
                 assert k in ["print_supported_models"], f"Unknown argument {k}"
 
-    config = load_config(args.config_file_path)
-
     # Register default logging plugin
     register_logging_plugin(PyLoggingPlugin())
 
-    bots = {}
+    # Load bots from config
+    with open(args.config_file_path) as f:
+        config = json.load(f)
 
-    for text_model_spec in config.text_models:
-        model = text_model_spec.model
-        api_key = text_model_spec.API_KEY
-        if (
-            text_model_spec.endpoint_account_override is not None
-            or text_model_spec.endpoint_model_override is not None
-        ):
-            _, account, _, model = model.split("/")
```

Comment:
I see that the sub-bots are registered under `/accounts/{account}/models/{model}`. What was the motivation behind it? Note that there are bots (like the function calling bot) that depend on multiple models. It feels a bit strange to elevate one of them, but not the others, to the url. Wouldn't it be cleaner to registed the bot under its config name instead?

Reply:
The vast majority of bots are a pass-through to Fireworks models, and we want to make it as easy as possible to stand up simple bots on top of our models. Special cases like function calling or CreativeQR can use `SERVER_endpoint_{account,model}_override` to customize the endpoint URI